src/io/byte_format.cr

```diff
@@ -43,8 +43,6 @@ module IO::ByteFormat
   abstract def encode(int : UInt64, io : IO)
   abstract def encode(int : Int128, io : IO)
   abstract def encode(int : UInt128, io : IO)
-  abstract def encode(int : Float32, io : IO)
-  abstract def encode(int : Float64, io : IO)
```

Comment:
Why are these useless?

Reply:
Because they are defined as non-abstract a few lines below. It doesn't make sense to define a method as abstract and as non-abstract in the same type.

Reply:
Whoops, I'm blind :)